HISTORY.md

```diff
@@ -1,10 +1,73 @@
 # History
 
+## 1.0.0 - 2022-4-25
+
+The main update of this release is the introduction of a `config`, which describes the `sdtypes` and `transformers` that will be used by the `HyperTransformer`
+for each column of the data. The user can interact with this config through the newly created methods `update_sdtypes`, `get_config`, `set_config`
+`update_transformers`, `update_transformers_by_sdtype`.
+
+This release also included various new features and updates, including:
+
+* Users can now transform subsets of the data using its own methods, `transform_subset` and `reverse_transform_subset`.
+* User validation was added for the following methods: `transform`, `reverse_transform`, `update_sdtypes`, `update_transformers`, `set_config`.
+* Unnecessary warnings were removed from `GaussianNormalizer.fit` and `FrequencyEncoder.transform`.
+* The user can now set a transformers as None.
+* Transformers that cannot work with missing values will automatically fill them in.
+* Added support for additional datetime formats.
+* Setting  `model_missing_values = False` in a transformer was updated to keep track of the percentage of missing values, instead of producing data containing `nan`'s.
+* All parameters were removed from the `HyperTransformer`.
+* `*PIIAnonymizer` was renamed to `AnonymizedFaker`.
+
+### New Features
+
+* Create methods for getting and setting a config - Issue [#418](https://github.com/sdv-dev/RDT/issues/418) by @amontanez24
```

Comment:
I closed and tagged a bunch more issues. I think all of the issues from the individual transformers should also be added